modules/wp-rest/classes/elementor-post-meta.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Elementor\Modules\WpRest\Classes;
+
+use Elementor\Plugin;
+use Elementor\Utils;
+
+defined( 'ABSPATH' ) || exit;
+
+class ElementorPostMeta {
+
+	public function register() :void {
+		$post_types = get_post_types_by_support( 'elementor' );
+
+		foreach ( $post_types as $post_type ) {
+			register_meta($post_type, '_elementor_edit_mode', [
```

Comment:
```suggestion
			register_meta( $post_type, '_elementor_edit_mode', [
```
Check this file for Linting issues.